aw_watcher_window/macos_jxa.py

```diff
@@ -21,17 +23,20 @@ def compileScript():
     if script:
         return script
 
+    logger.debug("compiling and caching script")
+
     from OSAKit import OSAScript, OSALanguage
 
     scriptPath = os.path.join(
         os.path.dirname(os.path.realpath(__file__)), "printAppStatus.jxa"
     )
+
     with open(scriptPath) as f:
         scriptContents = f.read()
 
-        # remove shebang line
-        if scriptContents.split("\n")[0].startswith("#"):
-            scriptContents = "\n".join(scriptContents.split("\n")[1:])
+        # # remove shebang line
```

Comment:
do you remember why this was needed? I'm not running into any errors with this removed.